app/assets/stylesheets/modules.css.scss

```diff
@@ -23,6 +23,20 @@
   }
 }
 
+.resource-block {
+  background:transparent image_url("noun_116372.png") 90% 50% no-repeat;
+  background-size: 100px;
+
+  a {
+    &.text { background-color:#F5F5EB; background-color:rgba(245, 245, 235, 0.95); }
+    &.video { background-color:#EBF5EB; background-color:rgba(235, 245, 235, 0.95); }
+    &.tutorial { background-color:#EBEBF5; background-color:rgba(235, 235, 245, 0.95); }
+
+    .fa { opacity:0.3; }
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Colon after property should be followed by one space